changes/pr3330.yaml

```diff
@@ -0,0 +1,2 @@
+enhancement:
+  - 'Adds schedule filters for month end or month start and specific day  - [#3330](https://github.com/PrefectHQ/prefect/pull/3330)'
```

Comment:
FYI I think these need to be double quotes to get parsed correctly

Reply:
The description also is no longer accurate (`is_month_end` already is in prefect).